app/models/recommendation.rb

```diff
@@ -0,0 +1,4 @@
+class Recommendation < ActiveRecord::Base
+  belongs_to :recommendable, polymorphic: true
```

Comment:
Yeah I believe so. That may help with the naming also. I know we're planning to have other things later, but no need to implement it now - it doesn't feel like we're painting into a corner, but I could be wrong.